vochain/vochaininfo/vochaininfo.go

```diff
@@ -234,86 +266,49 @@ func (vi *VochainInfo) NPeers() int {
 	return ni.NPeers
 }
 
+// BlocksLastMinute returns the number of blocks synced during the last minute.
+func (vi *VochainInfo) BlocksLastMinute() float64 {
+	vi.lock.RLock()
+	defer vi.lock.RUnlock()
+	return vi.blocksMinute
+}
+
 // Start initializes the Vochain statistics recollection.
-// TODO: use time.Duration instead of int64
 func (vi *VochainInfo) Start(sleepSecs uint64) {
 	log.Infof("starting vochain info service every %d seconds", sleepSecs)
```

Comment:
```suggestion
        if (sleepSecs == 0) {
                sleepSecs = 1
        }
	log.Infof("starting vochain info service every %d seconds", sleepSecs)
```

Reply:
Umm, I don't think sleepSecs=0 should be a valid value. It is ok if it panics cause this is wrong.

Reply:
Maybe we can just set a panic("sleepSecs is zero") to better control the crash